src/sig_stfl/xmss/LICENSE

```diff
@@ -1,8 +1,18 @@
 ## License
 
-This XMSS reference implementation is Copyright (c) 2024 SandboxAQ and licensed under both the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt) and [MIT License](LICENSE-MIT).
+This XMSS reference implementation is Copyright (c) 2024 SandboxAQ and licensed under the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt) OR MIT License at your option.
```

Comment:
Mirroring the xmss.md file, shouldn't there also be reference to the original authors (and the(ir) CC0 license) in this file?

Reply:
Oh, yes, you're right. Let me add that.

Reply:
updated in https://github.com/open-quantum-safe/liboqs/pull/1697/commits/983e6acd65f71dd35963c0f0ac00b0f9bcff89c2